tests/acceptance/features/apiSearch/dateSearch.feature

```diff
@@ -39,7 +39,9 @@ Feature: date search
       | Mtime:yesterday    | /yesterday.txt  |                 | /today.txt      |                 |
       | Mtime:"this week"  | /today.txt      |                 | /lastWeek.txt   | /lastMont.txt   |
       | Mtime:"this month" | /today.txt      |                 | /lastMont.txt   |                 |
```

Comment:
```suggestion
      | Mtime:"this week"  | /today.txt      |                 | /lastWeek.txt   | /lastMonth.txt   |
      | Mtime:"this month" | /today.txt      |                 | /lastMonth.txt   |                 |
```
Is this typo? If yes, then searching using `Mtime:"this month"` might fail as well.

Reply:
yeah. :+1: